circle.yml

```diff
@@ -7,9 +7,13 @@ dependencies:
   override:
     - curl -s -O https://raw.githubusercontent.com/atom/ci/master/build-package.sh
     - chmod u+x build-package.sh
+    - composer --version
     - composer config -g github-oauth.github.com cc4a091c096e7d3cfe053c3f669fb840be60ab98
     - composer global require "squizlabs/php_codesniffer=*"
 
 machine:
   php:
     version: 7.0.4
+  environment:
+    COMPOSER_DISABLE_XDEBUG_WARN: 1
+    PATH: "$HOME/.config/composer/vendor/bin:$PATH"
```

Comment:
It's probably good idea to also force composer folder with COMPOSER_HOME env var. Otherwise [it tries to be smart](https://github.com/composer/composer/blob/d8d0ee5870010253c98a7db4d27b07cd17bd6617/src/Composer/Factory.php#L69-L81).